docs/commerce/3.x/making-payments.md

```diff
@@ -141,5 +235,24 @@ If you would like to pay with a different gateway than the one that is already s
 ```
 
 ::: warning
-During this process the cart or order is never in the user’s session. This means that they could be paying for a cart and also have a separate cart in their current session.
+During this process the cart or order is never in the user’s session; they could be paying for a cart even with a separate cart in their current session.
+:::
+
+## Partial Payments in the Control Panel
+
+Store managers can specify an order payment amount using any gateway that supports partial payments, regardless of whether partial payments are allowed at checkout.
+
+To make a partial payment in the Craft control panel:
+
+1. Navigate to **Commerce** → **Orders**.
+2. Choose a cart or order to view its edit page
+3. Select the **Transactions** tab and choose **Make Payment**.
+4. Select a gateway that supports partial payments to enter the payment details and amount:\
+    ![Store manager’s partial payment modal](./assets/partial-payment-modal.png)
+5. Choose **Pay** to process the payment.
+
+::: tip
+To more quickly identify partially-paid orders, a store manager may want to add the **Paid Status** column in the Orders table.
```

Comment:
@mattstein I think Brandon calls it the "Order index page" or "Order index table" and not "Order table"